iset.mm

```diff
@@ -111628,6 +111628,52 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) , CC )
       OUVHNXLAUVLUVMXNUVHLXLAUVGXTWLXOYAUVJULXPXQWJXH $.
   $}
 
+  ${
+    $d A a f k m $.  $d A a f k x y $.  $d B a f m $.  $d B a f x y $.
+    $d S f k x y $.  $d a f k m ph $.  $d ph x y $.
+    fsumcllem.1 $e |- ( ph -> S C_ CC ) $.
+    fsumcllem.2 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x + y ) e. S ) $.
+    fsumcllem.3 $e |- ( ph -> A e. Fin ) $.
+    fsumcllem.4 $e |- ( ( ph /\ k e. A ) -> B e. S ) $.
+    ${
+      fsumcl2lem.5 $e |- ( ph -> A =/= (/) ) $.
+      $( - Lemma for finite sum closures.  (The "-" before "Lemma" forces the
+         math content to be displayed in the Statement List - NM 11-Feb-2008.)
```

Comment:
Why does this "- Lemma" have a special treatment ? Is it necessary to create a third category of "half lemma" ?

Reply:
I don't know (this was already there in set.mm) but I suppose because it is used in multiple theorems. Probably we should just call it a non-lemma.

Probably for another day, though, I'm a bit reluctant to try to come up with the right wording in this pull request.